lib/libc/Kconfig

```diff
@@ -138,6 +138,13 @@ config MINIMAL_LIBC_LL_PRINTF
 	  Build with long long printf enabled. This will increase the size of
 	  the image.
 
+config MINIMAL_LIBC_OPTIMIZE_STRING_FOR_SIZE
+	bool "Use size optimized string functions"
+	default n
```

Comment:
By convention, we do not specify `default n` because it is implied.

Reply:
Done via the comment below.